packages/ui/src/themes/color.styled.ts

```diff
@@ -41,16 +41,23 @@ export const colors = {
     warn: '#F1AE4A',
     error: '#FF624C',
     success: '#51C61A',
+    white: '#ffffff',
+    goldenrod: '#daa520',
   },
   background: {
     progressBar: '#1F1915',
     input: '#27221D',
     inputSecondary: '#3C3937',
     blur: 'rgba(29, 25, 23, 0.4)',
     separator: '#39322C',
-    separatorSecondary: '#272320',
```

Comment:
Why is this removed

Reply:
I had added this before, now renamed it to dropdown

Reply:
But it's used in other files, example: 
![image](https://github.com/Cypherock/cypherock-cysync/assets/80461749/0b0d647e-0be9-4755-944f-b08ce4f8f860)

Reply:
renamed back to separatorSecondary